Thai/travel.md

```diff
@@ -8,4 +8,6 @@ What do you want to do?
  
  [Have Tom Yum Kung. The food is as spicy as the weather.](Cuisine/TomYumKung.md)
 
- I want to sleep on a beach and dream of [hiking](Activities/Hiking.md) a mountain in the Northern of Thailand.
\ No newline at end of file
+ I want to sleep on a beach and dream of [hiking](Activities/Hiking.md) a mountain in the Northern of Thailand.
+ 
+ I want to go canoeing at Krabi and Trang [canoeing](Activities/canoeing.md). Its very fun.
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.